docs/source/index.rst

```diff
@@ -136,9 +136,9 @@ messy situations in everyday problems.
 
    install.rst
    10-minutes-to-dask.rst
+   deploying.rst
```

Comment:
I don't think the current deployment docs should live in "Getting Started". For new users cluster deployment is going to be a second step and more advanced topic. I think I would prefer for this to stay under the "Fundamentals" heading.

Reply:
I tend to agree but I can imagine the 10 minutes to dask doing a much better job of introducing deployment options

Reply:
This page is, I think, of very high value, especially for folks who are coming into Dask from a laptop perspective, but from a "I have a big dataset to deal with" perspective.  Right now if I go to the docs with that question in mind I don't see anything that I can click on that will take me closer to that information.

Reply:
> I tend to agree but I can imagine the 10 minutes to dask doing a much better job of introducing deployment options

I'm not seeing deployment options in that page?

I think that people arrive at docs asking themselves questions like "How do I run this thing on Kubernetes?".  I don't think that we currently serve these readers.

Reply:
Yeah there is a mention under "Scheduling" but to your point maybe that isn't framed in the way that people are asking it.

Reply:
Yeah, as someone who knows Dask really well and who goes to the docs regularly I haven't yet successfully navigated to the deploy page.  It isn't natural to me to look at scheduling (which I view as a more technical topic) in order to learn how to set it up.  I would expect this more under something like "install".